docs/reference/esql/functions/binary.asciidoc

```diff
@@ -2,19 +2,91 @@
 [[esql-binary-operators]]
 === Binary operators
 
-These binary comparison operators are supported:
+[[esql-binary-operators-equality]]
+==== Equality
+[.text-center]
+image::esql/functions/signature/equals.svg[Embedded,opts=inline]
+
+Supported types:
+
+include::types/equals.asciidoc[]
+
+==== Inequality `!=`
+[.text-center]
+image::esql/functions/signature/not_equals.svg[Embedded,opts=inline]
 
-* equality: `==`
-* inequality: `!=`
-* less than: `<`
-* less than or equal: `<=`
-* larger than: `>`
-* larger than or equal: `>=`
+Supported types:
 
-And these mathematical operators are supported:
+include::types/not_equals.asciidoc[]
 
+==== Less than `<`
+[.text-center]
+image::esql/functions/signature/less_than.svg[Embedded,opts=inline]
+
+Supported types:
+
+include::types/less_than.asciidoc[]
+
+==== Less than or equal to `<=`
+[.text-center]
+image::esql/functions/signature/less_than_or_equal.svg[Embedded,opts=inline]
+
+Supported types:
+
+include::types/less_than_or_equal.asciidoc[]
+
+==== Greater than `>`
+[.text-center]
+image::esql/functions/signature/greater_than.svg[Embedded,opts=inline]
+
+Supported types:
+
+include::types/greater_than.asciidoc[]
+
+==== Greater than or equal to `>=`
+[.text-center]
+image::esql/functions/signature/greater_than_or_equal.svg[Embedded,opts=inline]
+
+Supported types:
+
+include::types/greater_than_or_equal.asciidoc[]
+
```

Comment:
Shouldn't we add a header for "Mathematical operators" here? Or is it fine grouping these operators under "Binary operators"?

Reply:
Depends on how you want to classify them. I'm ok just calling them all "binary" operators because we don't have that many in the language. If you think separating them out is important I can though.

Reply:
If you're happy with "binary" then I'm too :)